vendor/assets/javascripts/neat/collection_editor.coffee

```diff
@@ -142,7 +143,7 @@ class window.Neat.CollectionEditor extends Backbone.View
       @views.push(view)
 
       $el = $(view.render().el)
-      $el.toggleClass 'alt', !(alt = !alt)
+      $el.toggleClass 'alt', !(alt = !alt) if @alternateRows
```

Comment:
What if it always added `alt` but you just didn't use CSS to style it?

Or does it conflict with something else?